spec/services/github_handler_spec.rb

```diff
@@ -48,23 +49,35 @@
       end
 
       it 'does not duplicate same user with different login' do
-        user
         expect {
-          github_handler.create_or_find_user(payload['requested_reviewer'])
+          github_handler.create_or_find_user(payload.requested_reviewer)
         }.to change(User, :count).by(0)
       end
     end
 
     describe 'pull request not existing' do
       it 'adds pr to the DB' do
         expect {
-          github_handler.create_or_find_pr(payload['pull_request'])
+          github_handler.opened
         }.to change(PullRequest, :count).by(1)
       end
     end
 
+    describe 'pull request close' do
+      let!(:pull_request) { create :pull_request, github_id: 1000 }
+
+      before do
+        github_handler.closed
+        pull_request.reload
+      end
+
+      it 'sets status closed' do
+        expect(pull_request.closed?).to be true
```

Comment:
`expect { github_handler.closed }.to change { pull_request.reload.closed? }.from(false).to(true)`